### ISSUE TITLE
p.259 원형 큐 디자인 Design Circular Queue 질문

### ISSUE THREAD

**Issue by DoTheBestMayB** [onlybooks/algorithm-interview#105]
p. 263 답안에서

isEmpty 와 isFull 함수에서 `self.p1 == self.p2` 는 없어도 되지 않나요?

isEmpty 인지는 front 포인터가 None 인지, isFull 인지는 end가 not None 인지만 확인하면 되므로
아래 코드처럼 간단하게 표현할 수 있다고 생각해요

`
    def isEmpty(self) -> bool:
        return self.q[self.front] is None

    def isFull(self) -> bool:
        return self.q[self.end] is not None
`

**Comment by likejazz** [comment]
네, 그렇네요. 굳이 불필요한 부분을 좀 더 제거할 수 있을 것 같습니다. 알려주셔서 감사합니다. 이 곳에 반영하도록 하겠습니다.
